stm32mp1.mk

```diff
@@ -204,6 +232,13 @@ BR2_PACKAGE_HOST_GENIMAGE=y
 BR2_ROOTFS_POST_SCRIPT_ARGS="$(BREXT_GENIMAGE_CONFIG) $(BINARIES_PATH) $(BREXT_BOOTFS_OVERLAY)"
 BR2_ROOTFS_POST_IMAGE_SCRIPT=$(BREXT_BOARD_PATH)/post-image.sh
 
+ifeq ($(WITH_RPMB_TEST),y)
+# Use S30optee init.d script that runs tee-supplicant with as root
```

Comment:
For commit "stm32mp1: WITH_RPMB_TEST=y enables RPMB secure storage"
`s/with as/as/` here, then:

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`